### PR TITLE
Add performance test for release 11.1.n

### PR DESCRIPTION
**Pull request by ivy-lli**

### CODE REVIEW

**Review by alexsuter — APPROVED** [review]

File: docker/ivy/11.1.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM axonivy/axonivy-engine:nightly-11.1
+
+ADD --chown=ivy:ivy testProjects/10.0.0/Performance/target/performance-1.0.0-SNAPSHOT.iar /usr/lib/axonivy-engine/deploy/performance.iar
```
Comment (weissreto) [inline docker/ivy/11.1.0/Dockerfile:3]: 10.0.0 still compatible?

**Review by weissreto — APPROVED** [review]

File: docker/ivy/11.1.0/Dockerfile (hunk 1 above)
Comment (ivy-lli) [reply docker/ivy/11.1.0/Dockerfile:3]: I've started the build. But locally I could start this docker container.
Let's see what the build means.

Comment (alexsuter) [reply docker/ivy/11.1.0/Dockerfile]: should be compatible

Comment (ivy-lli) [reply docker/ivy/11.1.0/Dockerfile]: no something with business data and jest which is no longer part of 11.1.0, but is used by the 10.0.0 project.

**Pull request merged**